data/human/human missions.txt

```diff
@@ -5856,8 +5856,8 @@ mission "Quicksilver Mixup 2"
 		has "event: quicksilver mixup 2 available"
 	on offer
 		conversation
-			`As you walk through <origin>'s spaceport, somebody wearing office attire and carrying a tablet power walks up to you after ending a hurried conversation with another pilot.`
-			`	"Captain! I work for the postmaster's office. The regular pilot who carries misposted mail to <planet> and back couldn't make it this month, some mechanical issues with his ship, I'm told. Would you be willing to carry this month's load there by <day> and then return with the mail from <planet>? We'll pay you what he'd be paid: <payment>."`
+			`As you walk through <origin>'s spaceport, somebody wearing office attire and carrying a power tablet walks up to you after ending a hurried conversation with another pilot.`
+			`	"Captain! I work for the postmaster's office. The regular pilot who carries misposted mail to <planet> and back couldn't make it this month, due to some mechanical issues with his ship, or so I'm told. Would you be willing to carry this month's load there by <day> and then return with the mail from <planet>? We'll pay you what he'd be paid: <payment>."`
```

Comment:
I don't think there's anything wrong with the way this was, and I don't think this is an improvement. At best, this is just different, which I'm not inclined to support.

Reply:
Due to is good, and then [look here](https://github.com/endless-sky/endless-sky/pull/10238#discussion_r1657500142) for the reason "or so" was put in.

Reply:
I have to agree with Warp here. The sentence fragment is fine considering it's dialog and not narration. This doesn't feel like a necessary change.